module/zfs/spa_stats.c

```diff
@@ -722,21 +722,24 @@ spa_io_history_destroy(spa_t *spa)
  */
 
 typedef struct spa_mmp_history {
+	uint64_t	mmp_kstat_id;	/* unique # for updates */
 	uint64_t	txg;		/* txg of last sync */
 	uint64_t	timestamp;	/* UTC time of of last sync */
 	uint64_t	mmp_delay;	/* nanosec since last MMP write */
 	uint64_t	vdev_guid;	/* unique ID of leaf vdev */
 	char		*vdev_path;
 	uint64_t	vdev_label;	/* vdev label */
+	int64_t		io_error;	/* error status of MMP write */
```

Comment:
`io_error` should be of type `int` for consistency with other errnos.

Reply:
Done